src/Place/Canonical/DBALDuplicatePlaceRepository.php

```diff
@@ -147,4 +147,32 @@ public function addToDuplicatePlaces(PlaceWithCluster $clusterRecordRow): void
             ])
             ->execute();
     }
+
+    public function howManyPlacesAreToBeImported(): int
```

Comment:
Can you not reuse `getPlacesNoLongerInCluster` ?

Reply:
howManyPlacesAreToBeImported() is the opposite from howManyPlacesAreToBeImported()